ecs-cli/modules/command/cluster_app_test.go

```diff
@@ -690,6 +692,20 @@ func TestClusterDownWithoutForce(t *testing.T) {
 	}
 }
 
+func TestDeleteClusterPrompt(t *testing.T) {
+	readBuffer := bytes.NewBuffer([]byte("yes\ny\nno\n"))
+	reader := bufio.NewReader(readBuffer)
+	if err := deleteClusterPrompt(reader); err != nil {
+		t.Fatalf("Expected no error with prompt to delete cluster")
```

Comment:
(multiple occurrences) Can you change these to `t.Errorf` instead of `t.Fatalf`?  Then you can have all of the assertions run even if one fails.  Also, can you ensure that the prompt answer is captured in the error message here?  That will help know which of the three assertions is failing.